src/stubs.ts

```diff
@@ -0,0 +1,32 @@
+import { Component } from 'vue'
+
+// Stubbing occurs when in vnode transformer we're swapping
+// component vnode type due to stubbing either component
+// or directive on component
+
+// In order to be able to find components we need to track pairs
+// stub --> original component
+
+// Having this as global might feel unsafe at first point
+// One can assume that sharing stub map across mounts might
+// lead to false matches, however our vnode mappers always
+// produce new nodeTypes for each mount even if you're reusing
+// same stub, so we're safe and do not need to pass these stubs
+// for each mount operation
+const stubs: WeakMap<Component, Component> = new WeakMap()
```

Comment:
We can't clear it safely, because there could be two `mount`ed wrappers simultaneously and `stubs` is shared among them

However, **Weak**Map give us clearing for free in our case